komodo/symlink/suggester/cli.py

```diff
@@ -111,16 +130,28 @@ def suggest_symlink_configuration(args, repo):
         job_name=args.jobname,
     )
 
+    if dry_run:
+        print("Dry-run, not making this PR:")
+        print(f"title={msg}")
+        print(f"body={body}")
+        print(f"head_target={target_branch}")
+        print(f"base={args.git_ref}")
+        return None
     return repo.create_pull(title=msg, body=body, head=target_branch, base=args.git_ref)
 
 
 def main():
     args = _parse_args()
     repo = _get_repo(os.getenv("GITHUB_TOKEN"), args.git_fork, args.git_repo)
 
-    pull = suggest_symlink_configuration(args, repo)
-    if pull is not None:
-        print(pull.html_url)
+    if args.verbose:
+        logger.setLevel(logging.INFO)
+
+    pull = suggest_symlink_configuration(args, repo, dry_run=args.dry_run)
+    if pull is None:
+        sys.exit(0)
```

Comment:
What is this case for?

Reply:
I guess `suggest_symlink_configuration()` returns `None` for a dry run (line 139), and if that comes back, we want to exit. Can't speak for Håvard but personally I think exiting from `main` is probably clearer than exiting from the other function.

Reply:
I have typed the code some more, and added a comment here!

Reply:
It is basically a legit condition where there is nothing to do.